### PR TITLE
GetSubmission and GetSubmissions - Adding Prefer header and expand options in Examples

### PR DESCRIPTION
**Pull request by cristobal-buenrostro**
the purpose of the PR is;

to mention the prefer request header in both pages.
show example of how to use $expand options in GetSubmission.

### CODE REVIEW

File: api-reference/beta/api/educationassignment-get.md [hunk 1]
```diff
@@ -75,10 +77,6 @@ GET https://graph.microsoft.com/beta/education/classes/f4a941ff-9da6-4707-ba5b-0
 [!INCLUDE [sample-code](../includes/snippets/java/get-educationassignment-java-snippets.md)]
 [!INCLUDE [sdk-documentation](../includes/snippets/snippets-sdk-documentation-link.md)]
 
-# [Go](#tab/go)
-[!INCLUDE [sample-code](../includes/snippets/go/get-educationassignment-go-snippets.md)]
-[!INCLUDE [sdk-documentation](../includes/snippets/snippets-sdk-documentation-link.md)]
-
```
Comment (isvargasmsft) [inline api-reference/beta/api/educationassignment-get.md:81]: Why are we removing this tab? I believe these are automatically generated and we should not manually edit the tabs. 

Is there a reason for this change?

File: api-reference/v1.0/api/educationsubmission-get.md [hunk 2]
```diff
@@ -96,22 +95,220 @@ The following is an example of the response.
...
+### Example 2: Get submission with $expand options
+#### Request
+The following is an example of the request.
+
+<!-- {
+  "blockType": "request",
+  "name": "get_submission_expand"
+}-->
+```http
+GET https://graph.microsoft.com/beta/education/classes/59069eb2-2a09-4d90-bb19-2089cc69d613/assignments/80da1069-a635-4913-813f-d775a5470a8f/submissions/869369de-3e5a-89eb-6f2d-83cd88f860b5?$expand=*
```
Comment (isvargasmsft) [inline api-reference/v1.0/api/educationsubmission-get.md:148]: This is a v1 document and in the example you are listing a request to the beta endpoint. Please revise to v1.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hi Cristobal, 

Thank you very much for your work in this PR. Very good shape overall. 

Please take a look at 2 comments:
1. Please clarify the reason to remove the Go snippets. 
2. Please make sure the endpoint in the examples are consistent with the version of the file. 

Feel free to ping me on the side if you have questions. 

Here to help!

Isaac

File: api-reference/beta/api/educationassignment-get.md (hunk 1 above)
Comment (cristobal-buenrostro) [reply api-reference/beta/api/educationassignment-get.md:81]: I didn't remove that snipped probably pushed incorrectly or was a conflict.
Done, added back

File: api-reference/v1.0/api/educationsubmission-get.md (hunk 2 above)
Comment (cristobal-buenrostro) [reply api-reference/v1.0/api/educationsubmission-get.md:148]: done

**Review by isvargasmsft — APPROVED** [review]

**Pull request merged**